src/main/kotlin/no/nav/syfo/client/pdl/domain/PdlResponse.kt

```diff
@@ -43,28 +53,73 @@ data class PersonNavn(
 )
 
 data class Adressebeskyttelse(
-    val gradering: Gradering
+    val gradering: Gradering,
 ) : Serializable
 
 enum class Gradering : Serializable {
     STRENGT_FORTROLIG_UTLAND,
     STRENGT_FORTROLIG,
-    FORTROLIG
+    FORTROLIG,
+    UGRADERT, // Ugraderte personer kan også ha tomt felt
 }
 
 data class PdlError(
     val message: String,
     val locations: List<PdlErrorLocation>,
     val path: List<String>?,
-    val extensions: PdlErrorExtension
+    val extensions: PdlErrorExtension,
 )
 
 data class PdlErrorLocation(
     val line: Int?,
-    val column: Int?
+    val column: Int?,
 )
 
 data class PdlErrorExtension(
     val code: String?,
-    val classification: String
+    val classification: String,
 )
+
+fun PdlHentPerson.toPersonName(): String? {
+    val navn = this.hentPerson?.navn?.first()
```

Comment:
Kan vurdere å bruke firstOrNull, da slipper du ?-sjekkene under. Ala val navn = this.hentPerson?.navn?.firstOrNull() ?: return null